cli_reference/cli_by_example_content.adoc

```diff
@@ -251,14 +251,17 @@
 
 [options="nowrap"]
 ----
-  // Tag the current image for the image stream 'openshift/ruby' and tag '2.0' into the image stream 'yourproject/ruby with tag 'tip':
+  // Tag the current image for the image stream 'openshift/ruby' and tag '2.0' into the image stream 'yourproject/ruby with tag 'tip'.
```

Comment:
All the comments in the examples we have in Origin end in full stop and these examples were the only using colons.